src/main/java/uk/gov/hmcts/reform/bulkscan/payment/processor/service/servicebus/handler/PaymentMessageHandler.java

```diff
@@ -100,6 +102,14 @@ private void createPayment(CreatePaymentMessage paymentMessage, String messageId
                 messageId,
                 paymentMessage.envelopeId
             );
+        } catch (FeignException exc) {
+            log.error(
```

Comment:
Similar as with the CCD client - logging would have to be conditional. Also, best if the message stated that payment wasn't processed because of error.

Reply:
Let's revert this change as well, not related to functional tests